slides/lack-of-time.md

```diff
@@ -1,3 +1,14 @@
 ##  Lack Of Time
 
 You'll have to take it.
+
+Note:
+Henning
+
+Unfortunately there are no shortcuts here. Due to the complexity of the subject and
+the large number of choices this will take time.
+What you might be surprised to hear is that we think you will be spending most of your
+time learning about non technical things related to the framework.
+
+HELP: Want to say a bit more about the non-technical things that take time, but also
+do not want to give away what you will be talking about in the next section. Any ideas?
```

Comment:
something like: we actually believe that if you look in the right places around a framework you'll notice very quickly if it is an option for you or not.